stackit/internal/services/objectstorage/credentialsgroup/resource_test.go

```diff
@@ -0,0 +1,325 @@
+package objectstorage
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/stackitcloud/stackit-sdk-go/core/utils"
+	"github.com/stackitcloud/stackit-sdk-go/services/objectstorage"
+)
+
+type objectStorageClientMocked struct {
+	returnError              bool
+	createProjectExecuteResp *objectstorage.GetProjectResponse
+	getCredentialsGroupsResp *objectstorage.GetCredentialsGroupsResponse
+}
+
+func (c *objectStorageClientMocked) CreateProjectExecute(_ context.Context, _ string) (*objectstorage.GetProjectResponse, error) {
+	if c.returnError {
+		return nil, fmt.Errorf("get credentials groups failed")
```

Comment:
This should be `fmt.Errorf("create project failed")`, right?

Reply:
Yes, fixed